shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -278,6 +278,16 @@
         head = clipView;
         break;
       }
+      case opacity:
+        embedded_view.alpha = (*iter)->GetOpacityParams().GetAlphaF() * embedded_view.alpha;
+        if ((*iter)->GetOpacityParams().offset.get().isZero()) {
+          break;
+        }
+        CATransform3D transform =
```

Comment:
Why are we bundling a transform with the opacity vs. pushing a separate transform mutator?